README.md

```diff
@@ -209,7 +209,6 @@ wavefront_sender = WavefrontProxyClient(
    distribution_port=2878,
    tracing_port=30000,
    event_port=2878
-   internal_flush=2,
```

Comment:
did you mean to remove this (or maybe `event_port=2878` needed a comma)? if you are removing it, also remove `# a nondefault interval (2 seconds) for flushing data from the sender to the proxy. Default: 5 seconds` comment above.

Reply:
Meant to remove. This is no longer in the WavefrontProxyClient. Will make changes to comment